tedana/io.py

```diff
@@ -25,6 +25,28 @@
 RefLGR = logging.getLogger("REFERENCES")
 
 
+class CustomEncoder(json.JSONEncoder):
+    """Convert some types because of JSON serialization and numpy
+    incompatibilities
+
+    # noqa: E501
+    See here: https://stackoverflow.com/questions/50916422/python-typeerror-object-of-type-int64-is-not-json-serializable/50916741
```

Comment:
```suggestion
    See here: https://stackoverflow.com/q/50916422/2589328
```
I grabbed the share link.